src/components/Taxonomy/Summary/index.tsx

```diff
@@ -0,0 +1,273 @@
+    }
+  }
+
+  componentDidUpdate(prevProps: LoadedProps) {
+    if (
+      prevProps.dataNames !== this.props.dataNames &&
+      this.props.dataNames?.payload
+    ) {
+      this.loadingVis = true;
+      this._populateData(this.props.dataNames.payload);
+      this.loadingVis = false;
+    }
+  }
+
+  _handleFocus = (accession: string) => {
+    if (!this.loadingVis)
+      this.props.goToCustomLocation({
+        description: {
+          main: { key: 'taxonomy' },
+          taxonomy: { db: 'uniprot', accession },
+        },
+      });
+  };
+
+  _populateData = ({ metadata: data, names, children }: Payload) => {
+    const lineage = data.lineage.trim().split(/\s+/);
+    let root: TaxNode | null = null;
+    let currentNode: TaxNode | null = null;
+    for (const node of lineage) {
+      const newNode: TaxNode = {
+        name: names[node].short,
+        id: node,
+      };
+      if (currentNode) {
+        currentNode.children = [newNode];
+      } else {
+        root = newNode;
+      }
+      currentNode = newNode;
+    }
+    if (currentNode) {
+      currentNode.name = data.name.short || data.name.name || data.accession;
+      currentNode.hitcount = data?.counters?.proteins as number;
+
+      if (data.children) {
+        currentNode.children = data.children.map((id) => ({
+          name: names[id].short,
+          id,
+          hitcount: children?.[id]?.proteins,
+        }));
+      }
+      this.setState({ data: root, focused: `${data.accession}` });
+    }
+  };
+
+  _handleChange = (event: React.FormEvent) => {
+    const value = (event.target as HTMLSelectElement).value;
+    const {
+      customLocation: { description },
+    } = this.props;
+    this.props.goToCustomLocation({
+      search: {
+        entry_db: value,
+      },
+      description,
+    });
+  };
+
+  _isSelected = (currentDB: DBInfo) => {
+    const {
+      customLocation: {
+        search: { entry_db: db },
+      },
+    } = this.props;
+    return (
+      ((db as string) || 'all').toLowerCase() ===
+      currentDB.canonical.toLowerCase()
+    );
+  };
+
+  render() {
+    if (
+      this.props.dataNames?.loading ||
+      !this.props.dataNames?.payload?.metadata
+    )
+      return <Loading />;
+    const { metadata, names } = this.props.dataNames.payload;
+    const {
+      customLocation: {
+        search: { entry_db: db },
+      },
+    } = this.props;
+
+    return (
+      <div className={css('vf-stack', 'vf-stack--400')}>
+        <div className={css('vf-stack')}>
+          <table className={css('vf-table', 'left-headers')}>
+            <tbody>
+              <tr>
+                <td style={{ maxWidth: '50%' }}>Tax ID</td>
+                <td>
+                  <Accession accession={metadata.accession} title="Tax ID" />
+                </td>
+              </tr>
+              {metadata.rank && (
+                <tr>
+                  <td>Rank</td>
+                  <td className={css('text-up')}>{metadata.rank}</td>
```

Comment:
```suggestion
                  <td className={css('first-letter-cap')}>{metadata.rank}</td>
```

Makes more sense to me to capitalize the rank instead of upper-case it.